libpod/container_internal.go

```diff
@@ -557,7 +557,7 @@ func (c *Container) setupStorage(ctx context.Context) error {
 }
 
 func (c *Container) processLabel(processLabel string) (string, error) {
-	if !c.config.Systemd && !c.ociRuntime.SupportsKVM() {
+	if (c.config.Systemd != nil && !*c.config.Systemd) && !c.ociRuntime.SupportsKVM() {
```

Comment:
What if systemd is nil? It seems like we should treat that as false

Reply:
this should be !c.Systemd()